packages/block-editor/src/experiments.js

```diff
@@ -7,6 +7,7 @@ import { __dangerousOptInToUnstableAPIsOnlyForCoreModules } from '@wordpress/exp
  * Internal dependencies
  */
 import * as globalStyles from './components/global-styles';
+import { default as __experimentalOffCanvasEditor } from './components/off-canvas-editor';
```

Comment:
Can this just be `import __experimentalOffCanvasEditor from './components/off-canvas-editor';` ?

Reply:
no b/c we export default